### PR TITLE
DROOLS-2960 : User should be able to use Scenario Simulation asset with the distribution war

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2689
https://issues.jboss.org/browse/DROOLS-2960

@kkufova Ok so so I bundled the two tickets together so it is easier to test. You can use the war created from here for testing.

### CODE REVIEW

File: kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml [hunk 1]
```diff
@@ -88,6 +88,8 @@
   <inherits name="org.drools.workbench.screens.dtablexls.DroolsWorkbenchDecisionTableXLSEditorClient"/>
   <inherits name="org.drools.workbench.screens.scorecardxls.DroolsWorkbenchScoreCardXLSEditorClient"/>
   <inherits name="org.drools.workbench.screens.testscenario.DroolsWorkbenchTestScenarioEditorClient"/>
+  <inherits name="org.drools.workbench.screens.scenariosimulation.DroolsWorkbenchScenarioSimulationEditorAPI" />
```
Comment (manstis) [inline kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml]: You probably don't need to inherit API as Client (probably) inherits already.

File: kie-wb-parent/kie-wb-webapp/src/main/resources/org/kie/workbench/KIEWebapp.gwt.xml [hunk 2]
```diff
@@ -99,6 +99,8 @@
   <inherits name="org.drools.workbench.screens.dtablexls.DroolsWorkbenchDecisionTableXLSEditorClient"/>
   <inherits name="org.drools.workbench.screens.scorecardxls.DroolsWorkbenchScoreCardXLSEditorClient"/>
   <inherits name="org.drools.workbench.screens.testscenario.DroolsWorkbenchTestScenarioEditorClient"/>
+  <inherits name="org.drools.workbench.screens.scenariosimulation.DroolsWorkbenchScenarioSimulationEditorAPI" />
```
Comment (manstis) [inline kie-wb-parent/kie-wb-webapp/src/main/resources/org/kie/workbench/KIEWebapp.gwt.xml]: You probably don't need to inherit API as Client (probably) inherits already.

File: kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml (hunk 1 above)
Comment (Rikkola) [reply kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml]: True

File: kie-wb-parent/kie-wb-webapp/src/main/resources/org/kie/workbench/KIEWebapp.gwt.xml (hunk 2 above)
Comment (Rikkola) [reply kie-wb-parent/kie-wb-webapp/src/main/resources/org/kie/workbench/KIEWebapp.gwt.xml]: True

**Comment by Rikkola** [comment]
jenkins please restart this

**Comment by Rikkola** [comment]
@manstis Thank you for the comments. Updated the PR.

**Comment by Rikkola** [comment]
This PR might need https://github.com/kiegroup/drools-wb/pull/935 to pass.

**Comment by Rikkola** [comment]
Rest fails to startup.

jenkins please restart this

**Comment by Rikkola** [comment]
restart? Looks like I like to copypaste.

jenkins please retest this.

**Comment by Rikkola** [comment]
FixNotMavenizedArtifactInfo.java missing a method. Most likely the jenkins builds are out of sync.

jenkins please retest this.

**Comment by Rikkola** [comment]
mortgage-process failed

jenkins please retest this.

**Comment by Rikkola** [comment]
While this is green, due to few missing features the plan is to add the editor in in the next sprint/release. So please do not merge this one :)

**Comment by Rikkola** [comment]
Looks like the git cloning failed.

jenkins please retest this.

**Comment by Rikkola** [comment]
org.kie.wb.selenium.ui.ProjectLibraryIntegrationTest.importAndBuildProjectFromStockRepository
    org.kie.wb.test.rest.functional.ProjectIntegrationTest.org.kie.wb.test.rest.functional.ProjectIntegrationTest
    org.kie.wb.test.rest.security.ProjectAccessIntegrationTest.org.kie.wb.test.rest.security.ProjectAccessIntegrationTest

jenkins please retest this.

**Comment by Rikkola** [comment]
org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testLibraryPerspectiveWithHeader
jenkins please retest this.

**Comment by Rikkola** [comment]
@manstis Good afternoon. Green and ready to go in.

**Pull request merged**